src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const item of actions) {
+    let itemType;
+
+    switch (item.type) {
+      case 'addProperties':
+        itemType = item.extraData;
+
+        Object.assign(state, itemType);
+        break;
+
+      case 'removeProperties':
+        itemType = item.keysToRemove;
```

Comment:
Better to use this object property instead of variable
(you didn't use it outside switch/case, you store different values in it)
It's harder to read your code with it